torch/_dynamo/output_graph.py

```diff
@@ -999,30 +999,38 @@ def cleanup_graph(self):
                     self.graph.erase_node(node1)
                     self.graph.erase_node(node2)
 
-    def get_graph_sizes_log_str(self, name):
-        graph_sizes_str = "TRACED GRAPH TENSOR SIZES\n"
-        graph_sizes_str += f"===== {name} =====\n"
+    def get_graph_sizes(self, *, structured, name=None):
```

Comment:
Maybe mypy is confused about the typing of the structured param? Is it a boolean?
```suggestion
    def get_graph_sizes(self, *, structured : bool, name : Optional[str] =None):
```